sky/sdk/example/game/assets/game_ui.json

```diff
@@ -0,0 +1,147 @@
+{"frames": [
+
+{
+	"filename": "bar_shield.png",
```

Comment:
This is an auto-generated file, by a texture packer. It's not meant to be edited manually.